src/frontend/src/pages/AdminDashboard/AdminDashboard.jsx

```diff
@@ -0,0 +1,95 @@
+import { Collaborate, Roadmap, Task, IbmWatsonTextToSpeech } from '@carbon/icons-react';
+import { Grid, Column, ModalWrapper } from '@carbon/react';
+import { ManagementTile, StatisticBox } from 'components';
+
+export default function AdminDashboard() {
+    return (
+        <Grid className='admin-page'>
+            <Column lg={16} md={8} sm={4} className='admin-page__r1'>
+                <h1 className='admin-page__heading'>Admin dashboard</h1>
+                <p className='admin-page__p'>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestiae harum quo
+                    culpa, quis labore nisi sint tempore illum reiciendis maiores enim! Aperiam
+                    obcaecati quo repellendus cum at! Soluta, ducimus iusto!
+                </p>
+            </Column>
+            <Column lg={16} md={8} sm={4} className='admin-page__r2'>
+                <StatisticBox
+                    title='Assigned groups'
+                    label='Are active groups forming?'
+                    value={12}
+                    changeInValue={0.3}
+                    valueIcon={<Collaborate size={24} />}
+                    variant='green'
+                />
+                <StatisticBox
+                    title='PDJ contributions'
+                    label='Are projects progressing?'
+                    value={15}
+                    changeInValue={-0.07}
+                    valueIcon={<Roadmap size={24} />}
+                    variant='blue'
+                />
+                <StatisticBox
+                    title='ILC journal activity'
+                    label='Are students reflecting?'
+                    value={15}
+                    valueIcon={<IbmWatsonTextToSpeech size={24} />}
+                    changeInValue={0.86}
+                />
+            </Column>
+            <Column lg={16} md={8} sm={4} className='admin-page__r3'>
+                <ManagementTile
+                    title='Groups'
+                    description='Archive, and lock groups in bulk'
+                    icon={<Collaborate size={48} />}
+                >
+                    <ModalWrapper
+                        buttonTriggerText='Add new owner contacts'
+                        modalHeading='Create new contacts'
+                    >
+                        {/* <PendingApplications groupApplications={groupApplication} /> */}
```

Comment:
This is a PR to setup the UI, theming, colours, etc. and skeleton out the layouts with routing. JS wiring + API work will come later